Gemfile

```diff
@@ -4,29 +4,28 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.22'
-gem 'rack-cache', '< 1.3.0'
+gem 'rails', '4.1.5'
```

Comment:
It can be newer than that, I pinned it to 4.1.5 though because it's what the SCL offered - https://www.softwarecollections.org/repos/rhscl/rh-ror41/epel-6-x86_64/rh-ror41-rubygem-rails-4.1.5-3.el6/ . I can relax it a bit `~> 4.1`, 4.2 would bring some incompatibilities.